relayer/logger.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+
+logger = logging.getLogger('relayer')
+
+
+def log_kafka_message(topic, payload, partition_key=None):
+    logger.debug('Writing to kafka topic: %s, partition_key: %s, message: %s', topic, partition_key, payload)
```

Comment:
`Writing to kafka` seems a bit verbose/redundant, we may be good with just `'topic: %s, ...`